resources/RegexGrammar.pp

```diff
@@ -42,11 +42,13 @@
 //
 
 // Character classes.
+%token  negative_class_fc_       \[\^(?=\])         -> class_fc
```

Comment:
what does "fc" stand for?

Reply:
first character - I know it is hard to understand, but should full/long name be used which will break the indentation?

if yes, feel free to propose changes

Reply:
maybe at least add a comment at the end of the line

Reply:
added